### PR TITLE
Release v0.23.0

### PR DESCRIPTION
**Pull request by alindima**
Changes automatically generated with `devtool prepare-release`.

## Reason for This PR

`[Author TODO: add issue # or explain reasoning.]`

## Description of Changes

`[Author TODO: add description of changes.]`

- [ ] This functionality can be added in [`rust-vmm`](https://github.com/rust-vmm).

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [ ] All commits in this PR are signed (`git commit -s`).
- [ ] The reason for this PR is clearly provided (issue no. or explanation).
- [ ] The description of changes is clear and encompassing.
- [ ] Any required documentation changes (code and docs) are included in this PR.
- [ ] Any newly added `unsafe` code is properly documented.
- [ ] Any API changes are reflected in `firecracker/swagger.yaml`.
- [ ] Any user-facing changes are mentioned in `CHANGELOG.md`.
- [ ] All added/changed functionality is tested.

### CODE REVIEW

**Review by acatangiu — APPROVED** [review]

**Review by acatangiu — APPROVED** [review]

**Review by sandreim — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## [Unreleased]
+## [0.23.0]
```
Comment (acatangiu) [inline CHANGELOG.md:3]: We need a `Full AMD support` in `Added` section.

**Review by acatangiu — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (alindima) [reply CHANGELOG.md:3]: Done

**Comment by alindima** [comment]
@sandreim @acatangiu PTAL

**Review by acatangiu — APPROVED** [review]

File: CREDITS.md [hunk 2]
```diff
@@ -106,21 +131,27 @@ Contributors to the Firecracker repository:
 * Rolf Neugebauer <neugebar@amazon.com>
 * Sam Jackson <sam@clique.app>
 * Samuel Ortiz <sameo@linux.intel.com>
+* sandreim <54316454+sandreim@users.noreply.github.com>
```
Comment (raduiliescu) [inline CREDITS.md:134]: This looks to be an error.

File: CHANGELOG.md [hunk 3]
```diff
@@ -30,6 +30,7 @@
 - Added a new API call, `PUT /snapshot/load`, for loading a snapshot.
 - Added new jailer command line argument `--cgroup` which allow the user to
   specify the cgroups that are going to be set by the Jailer.
+- Added full support for AMD CPUs (General Availability).
```
Comment (raduiliescu) [inline CHANGELOG.md:33]: I would also add more details or a reference to our readme section where we say what AMD cpus we support

File: CREDITS.md (hunk 2 above)
Comment (alindima) [reply CREDITS.md:134]: why exactly? I think @sandreim chose to not expose his email on github

Comment (raduiliescu) [reply CREDITS.md:134]: His address, the correct one, is above

Comment (alindima) [reply CREDITS.md:134]: will fix it

File: CHANGELOG.md (hunk 3 above)
Comment (alindima) [reply CHANGELOG.md]: done. added a link to the supported platforms

File: CREDITS.md (hunk 2 above)
Comment (alindima) [reply CREDITS.md]: done

**Review by raduiliescu — APPROVED** [review]

**Review by acatangiu — APPROVED** [review]

File: .mailmap [hunk 4]
```diff
@@ -11,3 +11,7 @@ Tamio-Vesa Nakajima <tamiove@amazon.com> <tnk@amazon.com>
 Iulian Barbu <iul@amazon.com>
 Petre Eftime <epetre@amazon.com> <petre.eftime@gmail.com>
 karthik nedunchezhiyan <karthik.n@zohocorp.com> <karthik1705.n@gmail.com>
+Alin Dima <alindima@amazon.com>
+Andrei Sandu <sandreim@amazon.com> <54316454+sandreim@users.noreply.github.com>
```
Comment (sandreim) [inline .mailmap:15]: I think we should remove the bogus <54316454+sandreim@users.noreply.github.com> email address.

Comment (alindima) [reply .mailmap:15]: if we do remove it, you will appear twice in the credits.md file, once for your real email address and once for this new one

**Comment by raduiliescu** [comment]
I think this change should also go in master

**Comment by acatangiu** [comment]
> I think this change should also go in master

Yes, post-release, the release branch should be merged to mainline. Either through `merge` or through manual `cherry-pick` of missing commits.

**Comment by alindima** [comment]
Indeed, I will cherry-pick the relevant commits

**Pull request merged**